frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -129,7 +129,7 @@ const getAvailableFilters = (initialFilters) => {
 };
 
 const filterGroupNameMap = {
-  kind: 'Type',
+  kind: 'TYPE',
```

Comment:
@serenamarie125 this change makes the two pages unbalanced is what @christianvogt is saying.

Today (before this change) this is the two views:

![Screen Shot 2020-01-10 at 3 18 33 PM](https://user-images.githubusercontent.com/8126518/72183760-c9c1a200-33bc-11ea-82b2-7f0c1b91ecb1.png)
![Screen Shot 2020-01-10 at 3 18 58 PM](https://user-images.githubusercontent.com/8126518/72183761-ca5a3880-33bc-11ea-9b68-ea1ed0bf1666.png)

So changing it to all capitals makes it not match the Operator Hub (and the colour / spacing is different than the designs)